test/jasmine/tests/mapbox_test.js

```diff
@@ -1487,6 +1548,11 @@ describe('mapbox plots', function() {
                 var s = d3SelectAll('.mapboxgl-ctrl-attrib');
                 expect(s.size()).toBe(1);
                 expect(s.text()).toEqual('© Mapbox © OpenStreetMap Improve this map');
+                assertLinks(s, [
+                    'https://www.mapbox.com/about/maps/',
+                    'http://www.openstreetmap.org/about/',
```

Comment:
@alexcjohnson any suggestion on how to replace this `http` link with `https`?

Reply:
if this is coming from MapboxGL.js let's just leave it for now

Reply:
Yeah. It's coming from there.